### PR TITLE
fix(docs): unit-test.mdx to work

### PR DESCRIPTION
**Pull request by kazuooooo**
## I have read the [CONTRIBUTING.md](https://github.com/supabase/supabase/blob/master/CONTRIBUTING.md) file.

YES

## What kind of change does this PR introduce?
docs update

## What is the current behavior?
The contents of [Testing your Edge Functions](https://supabase.com/docs/guides/functions/unit-test) do not work

## What is the new behavior?
I have modified the contents of the document so that they work correctly

## Additional context

Add any other context or screenshots.

### CODE REVIEW

File: apps/docs/content/guides/functions/unit-test.mdx [hunk 1]
```diff
@@ -40,11 +40,12 @@ import {
   assertEquals,
 } from 'https://deno.land/std@0.192.0/testing/asserts.ts'
 import { createClient, SupabaseClient } from 'https://esm.sh/@supabase/supabase-js@2.23.0'
-import { delay } from 'https://deno.land/x/delay@v0.2.0/mod.ts'
```
Comment (kazuooooo) [inline apps/docs/content/guides/functions/unit-test.mdx:43]: `delay` is not used

File: apps/docs/content/guides/functions/unit-test.mdx [hunk 2]
```diff
@@ -40,11 +40,12 @@ import {
...
+import { load } from 'https://deno.land/std@0.224.0/dotenv/mod.ts'
 
 // Set up the configuration for the Supabase client
-const supabaseUrl = Deno.env.get('SUPABASE_URL') ?? ''
-const supabaseKey = Deno.env.get('SUPABASE_ANON_KEY') ?? ''
+const env = await load()
+const supabaseUrl = env['SUPABASE_URL'] ?? ''
+const supabaseKey = env['SUPABASE_ANON_KEY'] ?? ''
```
Comment (kazuooooo) [inline apps/docs/content/guides/functions/unit-test.mdx:48]: I needed to use load to get Environment variables.
Ref: https://docs.deno.com/runtime/manual/basics/env_variables#env-file

File: apps/docs/content/guides/functions/unit-test.mdx [hunk 3]
```diff
@@ -135,19 +136,19 @@ To locally test and debug Edge Functions, you can utilize the Supabase CLI. Let'
 
    ```bash
    # creates the file
-   touch .env.local
+   touch .env
```
Comment (kazuooooo) [inline apps/docs/content/guides/functions/unit-test.mdx:139]: I needed to create `.env` file not `.env.local` file

File: apps/docs/content/guides/functions/unit-test.mdx [hunk 4]
```diff
@@ -135,19 +136,19 @@ To locally test and debug Edge Functions, you can utilize the Supabase CLI. Let'
...
    # Alternatively, you can open it in your editor:
-   open .env.local
+   open .env
    ```
 
 4. To run the tests, use the following command in your terminal:
 
    ```bash
-   deno test --allow-all supabase/functions/function-one-test.ts --env-file .env.local
+   deno test --allow-all supabase/functions/function-one-test.ts
```
Comment (kazuooooo) [inline apps/docs/content/guides/functions/unit-test.mdx:151]: Original command raise a error.
```
error: unexpected argument '--env-file' found
```

I needed to delete `--env-file` option

**Review by charislam — APPROVED** [review]
Thanks @kazuooooo!

**Pull request merged**